awx/main/credential_plugins/hashivault.py

```diff
@@ -47,15 +47,20 @@
             'multiline': False,
             'help_text': _('Name of the namespace to use when authenticate and retrieve secrets'),
         },
+        {
+            'id': 'kubernetes_role',
+            'label': _('Kubernetes role'),
+            'type': 'string',
+            'multiline': False,
+            'help_text': _('The Role for Kubernetes Authentication'),
```

Comment:
I'm wondering if we should include an example or point to some documentation here. It's not immediately clear to me what is expected for this field.

Reply:
@shanemcd good idea, added.